R/length.R

```diff
@@ -62,7 +62,7 @@
 #'   length = c(10, 8)
```

Comment:
Documentation for `@param domain` should be changed above as it is inherited to other functions

```r
#' @param domain Appropriate CDSIC dataset name, e.g. ADAE, DM. Used to subset
#'   the metadata object. If none is passed, then [xportr_domain()] or [xportr_metadata()]
#'   must be called before hand to set the domain as an attribute of `.df`.
```

Reply:
Good catch! I thought it was weird we weren't documenting this anywhere.